CorePluginFramework/test/unit/src/au/gov/asd/tac/constellation/plugins/parameters/types/FileParameterTypeNGTest.java

```diff
@@ -18,11 +18,15 @@
 import au.gov.asd.tac.constellation.plugins.parameters.PluginParameter;
 import au.gov.asd.tac.constellation.plugins.parameters.types.FileParameterType.FileParameterKind;
 import au.gov.asd.tac.constellation.plugins.parameters.types.FileParameterType.FileParameterValue;
+import static au.gov.asd.tac.constellation.plugins.parameters.types.FileParameterType.getFileFilters;
 import au.gov.asd.tac.constellation.utilities.file.FileExtensionConstants;
+import com.jogamp.newt.event.KeyEvent;
```

Comment:
Thoguht this might need to be `javafx.scene.input.KeyEvent` but it seems its actually unused and can be removed.